pkg/infra/grn/grn.proto

```diff
@@ -17,4 +17,8 @@ message GRN {
 	// ResourceIdentifier is used by the underlying service to identify the
 	// resource.
 	string ResourceIdentifier = 4;
+
+	// The group represents the specific resource group the resource belongs to.
+	// This is a unique value for each plugin and maps to the k8s Group
+	string ResourceGroup = 5;
```

Comment:
should we move this so it is ordered #2?

Reply:
for sure -- if this was used anywhere we should not reorder it, but given that it is not yet used should we make the order more closely match usage?

Reply:
We'll have a larger question there when it comes to the various entity proto definitions since there are new fields being added there too, I'm tempted to merge this as-is and look at renumbering as a separate issue. wdyt?